drivers/at_cmd/at_cmd.c

```diff
@@ -143,13 +151,13 @@ static void socket_thread_fn(void *arg1, void *arg2, void *arg3)
 		item->callback = NULL;
 
 		bytes_read = recv(common_socket_fd, item->data,
-				  sizeof(item->data), 0);
+					sizeof(item->data), 0);
```

Comment:
Please don't modify proper code format. Here and in a few places below.

Reply:
Those were added automatically by my editor.  Do you have a reference for the appropriate style I can use with Sublime Text?  What editor do the Nordic folks use?

Reply:
We generally follow [Zephyr coding style guidelines](https://docs.zephyrproject.org/latest/contribute/index.html#coding-style) here, which in turn follows Linux kernel style. 

We have a `clang-format` config file in this repository, so you can run `git-clang-format` after you stage your changes to see if they comply. Keep in mind though, that this tool has some issues, unfortunately - for instance, I've experienced problems with structure formatting several times, so I needed to fix them manually.

I'm not familiar with Sublime, so I'm not able to comment if there're any format configs for it.
Nordic officially recommended IDE is Segger Embedded Studio. Personally, I use VS code,, have no issues with it.